pkg/authorization/bootstrap/policy.go

```diff
@@ -23,31 +23,32 @@ import (
 	rbacrest "k8s.io/kubernetes/pkg/registry/rbac/rest"
 	"k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy"
 
+	"github.com/kcp-dev/kcp/pkg/apis/core"
 	"github.com/kcp-dev/kcp/pkg/apis/tenancy"
 )
 
 const (
-	// SystemKcpClusterWorkspaceAccessGroup is a group that gives a user basic access to a workspace.
-	// It does not give them any permissions in the workspace.
-	SystemKcpClusterWorkspaceAccessGroup = "system:kcp:clusterworkspace:access"
-	// SystemKcpClusterWorkspaceAdminGroup is an admin group per cluster workspace. Members of this group have all permissions
-	// in the referenced cluster workspace (capped by maximal permission policy).
-	SystemKcpClusterWorkspaceAdminGroup = "system:kcp:clusterworkspace:admin"
 	// SystemKcpAdminGroup is global admin group. Members of this group have all permissions across all cluster workspaces.
 	SystemKcpAdminGroup = "system:kcp:admin"
 	// SystemKcpWorkspaceBootstrapper is the group used to bootstrap resources, both during the root setup, as well
 	// as when the default APIBinding initializing controller performs its bootstrapping for initializing workspaces.
 	// We need a separate group (not system:masters) for this because system-owned workspaces (e.g. root:users) need
 	// a workspace owner annotation set, and the owner annotation is skipped/not set for system:masters.
 	SystemKcpWorkspaceBootstrapper = "system:kcp:tenancy:workspace-bootstrapper"
+	// SystemLogicalClusterAdmin is a group used by the scheduler to create LogicalCluster resources.
+	// This group allows it to skip the entire authorization stack except the bootstrap policy authorizer.
+	// Otherwise, access to a top level org or a parent workspace would be required.
```

Comment:
Do we still have the concept of "top level org"?

Reply:
we don't.

Reply:
done